stripe.php

```diff
@@ -275,3 +268,19 @@ function stripe_civicrm_managed(&$entities) {
 
   return _stripe_civix_civicrm_managed($entities);
 }
+
+/**
+ * Implementation of hook_civicrm_alterContent
+ *
+ * @return void
+ */
+function stripe_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
+  if($context == 'form') {
+    $stripe_key = stripe_get_key($object);
+    if(empty($stripe_key)) {
+      return;
+    }
+    $stripeJSURL = CRM_Core_Resources::singleton()->getUrl('com.drastikbydesign.stripe', 'js/civicrm_stripe.js');
```

Comment:
Is this more reliable than addScriptFile()?

Have you seen this approach used in other CiviCRM core or contributed extensions?